pkg/mcs/exportspec.go

```diff
@@ -60,8 +61,32 @@ type GlobalProperties struct {
 const (
 	prefix                  = "lighthouse.submariner.io"
 	ServiceExportAnnotation = prefix + "/" + "serviceExportSpec"
+	NameField               = "name"
+	NamespaceField          = "namespace"
+	TypeField               = "type"
+	PortField               = "ports"
+	AffinityField           = "affinity"
+	AffinityConfigField     = "affinity config"
 )
 
+type CompatibilityError struct {
+	clusterID string
+	field     string
+}
+
+func (err *CompatibilityError) Error() string {
+	if err.field != NameField && err.field != NamespaceField {
+		return fmt.Sprintf("field %s conflicts with value in cluster %s",
+			err.field, err.clusterID)
+	}
+	return fmt.Sprintf("The same service has different %s in cluster %s",
```

Comment:
this is reached when the export specification refers to a different service (name.namespace)?
If so, the error message is confusing (e.g., `"the export refers to a different service and is incompatible"`).